### ISSUE TITLE
Global Application menu

### ISSUE THREAD

**Issue by GlennMm** [vars1ty/HybridBar#22]
I would like to request a feature. Can you guys add a global application menu if it possible just like the macos one or kde global menu

**Comment by vars1ty** [comment]
No, this is a status bar and not an application launcher. If you want something similar to Mac OS, you can probably customize wofi to behave similar enough to it.
